program/find-average-of-digits-of-a-number/FindAverageOfDigitsOfANumberByRecursion.kt

```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner
```

Comment:
@Coder-12-ux rename the file , you have used the same file name for both recursion and non-recursion